.github/workflows/_build-and-test-locally.yml

```diff
@@ -292,12 +292,18 @@ jobs:
       fail-fast: false
       matrix:
         pg_version: ${{ fromJson(inputs.pg-versions) }}
+        no_default_lfc: [lfc]
```

Comment:
A negation in a variable name makes it harder to read/understand, especially with negative cases like `no_default_lfc: nolfc`, let's use something positive, like `USE_LFC_BY_DEFAULT` (if we must state that this is a default)

Reply:
I agree, that readability suffers.
This code is removed due to refactoring